docs/guide/topics/embeds.rst

```diff
@@ -0,0 +1,421 @@
+- :attr:`Embed.footer`
+
+.. code-block:: python
+
+    >>> print(embed.thumbnail.proxy_url)
+    'https://media.discordapp.net/attachments/123456789012345678/8765432187654321/image.png'
+
+
+Other Methods
+-------------
+
+There are a few other methods that may be useful when working with embeds.
+
+.. method:: Embed.from_dict()
+    :noindex:
+    
+    Creates an embed from a Python dictionary.
+
+    .. code-block:: python
+
+        payload = {
+            "title": "Hello, World!",
+            "description": "This bot is running on discord.py!",
+            "color": 0x00FF00
+        }
+        embed = discord.Embed.from_dict(payload)
+
+    .. warning::
+        
+        Each key needs to match the structure of an embed from Discord's API.
+        Namely, the API doesn't alias `colour` and `color` needs to be used instead.
+        :ddocs:`Embeds as documented by Discord<resources/channel#embed-object>`.
+        
+
+.. method:: Embed.to_dict()
+    :noindex:
+
+    Inversely, `to_dict` converts the embed to a dictionary compatible with the API's embed specification.
+
+    .. code-block:: python
+
+        >>> embed.to_dict()
+        {'title': 'Hello, World!', 'description': 'This bot is running on discord.py!', 'type': 'rich', 'color': 65280}
+
+Other can be found in the :class:`Embed` documentation.
+
+
+Restrictions and limits
+------------------------
+
+There are a few restrictions and limits that you should be aware of when working with embeds.
+
+Markdown support
+~~~~~~~~~~~~~~~~~
+
+Markdown like **this**, *that*, and even ``this`` is supported in embeds, but only in certain fields.
+
+More about this can be found at :ref:`_guide_topic_markdown`.
+
+All strings
+~~~~~~~~~~~~
+
+All values passed to the embed must be a string.
+
+Except for ``timestamp`` and ``colour`` which must be a :class:`datetime.datetime` and :class:`Colour` / ``int``, respectively.
+
+discord.py attempts to convert all values given to string using ``str()``.
+This can be confusing for beginning users of Python as they may not be aware of this, most objects have a ``__str__`` method that 
+returns a string representation of the object. Most objects in discord.py also do this, like :class:`Asset`, that is returned from
+attributes like :attr:`User.avatar` and :attr:`Guild.icon`, calling ``str()`` on any of those returns the URL of the asset.
+That is precisely why you can pass these attributes to the ``url`` or ``icon_url`` parameter of the methods or :attr:`User` to ``name`` in
+:meth:`embed.set_author() <Embed.set_author>`. It will work but it's not encouraged to do so.
+
+Try it out:
+
+.. code-block:: python
+
+    embed.set_author(name=bot.user, icon_url=bot.user.display_avatar)
+
+Character limits
+~~~~~~~~~~~~~~~~~
+
+Each field has its own character limit. Unfortunately, listing all the limits here would quickly become 
+outdated, but you can find them in the :ddocs:`Discord API documentation<resources/channel#embed-object>`.
```

Comment:
```suggestion
outdated, but you can find them in the :ddocs:`Discord API documentation<resources/channel#embed-object-embed-limits>`.
```

The link supplied isn't the correct one as you would need to scroll down further, but this one should be the correct one: https://discord.com/developers/docs/resources/channel#embed-object-embed-limits